### PR TITLE
Improved paging user tree

### PR DESCRIPTION
**Pull request by IliaStepanov**

### CODE REVIEW

**Comment by IliaStepanov** [comment]
[[BE] User paging func](https://trello.com/c/o7GZLZjp/92-be-user-paging-func)

File: src/main/java/com/epam/lowcost/controller/UserController.java [hunk 1]
```diff
@@ -21,38 +24,47 @@
...
-        model.addAttribute("users", userService.getAllUsers());
+
+    @RequestMapping(value = USERS)
+    public String showUsers(Model model,Pageable pageable) {
+        model.addAttribute("users", userService.getAllUsers(pageable));
         return USERS_PAGE;
     }
 
+    @RequestMapping(value = PAGE, method = RequestMethod.GET)
+    public String setUsersByPage(@RequestParam String number, @RequestParam String fromPage, Model model) {
```
Comment (vakhonin) [inline src/main/java/com/epam/lowcost/controller/UserController.java]: When this method is used?

Comment (IliaStepanov) [reply src/main/java/com/epam/lowcost/controller/UserController.java]: Left it from previous realization, removed.

Comment (vakhonin) [reply src/main/java/com/epam/lowcost/controller/UserController.java]: c-c-c...

**Pull request merged**